t/t-migrate-import.sh

```diff
@@ -409,6 +409,30 @@ begin_test "migrate import (include/exclude ref with filter)"
 )
 end_test
 
+begin_test "migrate import (above)"
+(
+  set -e
+  setup_single_local_branch_untracked
+
+  md_main_oid="$(calc_oid "$(git cat-file -p "refs/heads/main:a.md")")"
+  txt_main_oid="$(calc_oid "$(git cat-file -p "refs/heads/main:a.txt")")"
+
+  git lfs migrate import --above 121B
+  # Ensure that 'a.md', whose size is above our 121 byte threshold
+  # was converted into a git-lfs pointer by the migration.
+  assert_local_object "$md_main_oid" "140"
+  assert_pointer "refs/heads/main" "a.md" "$md_main_oid" "140"
+  refute_pointer "refs/heads/main" "a.txt" "$md_main_oid" "120"
+  refute_local_object "$txt_main_oid" "120"
+
+  # The migration should have identified that *.md files are now
+  # tracked because it migrated a.md
+  main_attrs="$(git cat-file -p "$main:.gitattributes")"
+
+  echo "$main_attrs" | grep -q "*.md filter=lfs diff=lfs merge=lfs"
```

Comment:
Thanks so much for making the other changes above in this test!

The only other nice-to-have fixup I can see here would be to add a line after this one which checks that there is no `*.txt` entry in `.gitattributes`.  I believe some of the other tests do this [already](https://github.com/git-lfs/git-lfs/blob/1f4e688e32bedfa067bb34b88ca52995387b3d02/t/t-migrate-import.sh#L109) with a `-vq` line like:
```suggestion
  echo "$main_attrs" | grep -q "*.md filter=lfs diff=lfs merge=lfs"
  echo "$main_attrs" | grep -vq "*.txt filter=lfs diff=lfs merge=lfs"
```

Reply:
done